src/server_manager/electron_app/digitalocean_oauth.ts

```diff
@@ -96,16 +96,6 @@ function getAccount(accessToken: string): Promise<Account> {
   });
 }
 
-export interface OauthSession {
```

Comment:
We are no longer encoding the dependency of digitalocean_auth.ts on OauthSession.
Could you add a https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html to point to the new .d.ts file?

Reply:
We've been using `tsconfig.json` in lieu of triple-slash directives:
https://github.com/Jigsaw-Code/outline-server/blob/master/src/server_manager/electron_app/tsconfig.json#L16

I see benefits and drawbacks to both methods but I do like that `tsconfig.json` serves as a central place to add them. WDYT?

Reply:
The difference is that in those other cases we are still expressing the dependency with an import statement. Your change is for global definitions, and they have no import statement. The triple-slash would make the dependency clearer.

Reply:
@fortuna it seems triple-slashes are disallowed by the style guide:
* https://github.com/google/ts-style/blob/master/tslint.json#L34

It's this rule:
https://palantir.github.io/tslint/rules/no-reference/

I couldn't find much on best practices regarding `d.ts` for globals, specifically, but seems to me like any decent IDE nowadays will help you quickly locate the definition?

Reply:
I'm aware that /// is disallowed in the style guide, but so is defining global symbols like we are doing. We should be using modules instead, etc. We are in an exception territory, since that's how the preload script works.

I guess it's fine either way. I don't feel strongly